Source/Scene/OIT.js

```diff
@@ -554,12 +554,14 @@ define([
 
         for (j = 0; j < length; ++j) {
             command = commands[j];
+            command = defined(command.derivedCommands.logDepth) ? command.derivedCommands.logDepth.logDepthCommand : command;
```

Comment:
Sorry for the perhaps obvious question here, but is everything based on derived commands?  If so, why there is there so much custom code for models, ellipsoid primitives, etc.

Reply:
The custom code adds the function calls but the function bodies are surrounded by `#ifdef`s. The derived commands will add the defines when log depth is supported.